materials/section-0/README.md

```diff
@@ -0,0 +1,228 @@
+
+
+# Introduction
+
+What is Clojure?
+
+>Clojure was forged in a mythic volcano by Rich Hickey. Using an alloy of Lisp, functional programming, and a lock of his own epic hair, he crafted a language that’s delightful yet powerful. Its Lisp heritage gives you the power to write code more expressively than is possible in most non-Lisp languages, and its distinct take on functional programming will sharpen your thinking as a programmer.
+
+\- CLOJURE for the BRAVE and TRUE
+
+That is a nice and short summary of Clojure. Although, some key things are missing from the description. As the description points out, Clojure is a Lisp, it's also dynamically typed, and it runs on the JVM. In the next section, we'll go through the fundamentals of the language.
+
+## Table of contents
+
+1. [Syntax](#syntax)
+2. [Data Types](#data-types)
```

Comment:
I'm wondering if this should have a different name, since datatypes refers to something else in Clojure already, people might get confused.